EStimLibrary.sln

```diff
@@ -7,6 +7,10 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "src", "src", "{7E35BD38-108
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "EStimLibrary", "src\EStimLibrary\EStimLibrary.csproj", "{23710686-EDC8-41FC-AAC8-52B002F488D6}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "tests", "tests", "{3193A4CC-4C0F-4A71-BF98-B7EF41A15C5D}"
```

Comment:
Revert these changes to the EStimLibrary.sln. The tests/EstimLibrary.UnitTests/EstimLibrary.UnitTests.sln should already have the 2 projects included. EStimLibrary.sln is to open the library only